helper/include/odp/helper/ipsec.h

```diff
@@ -93,6 +94,18 @@ int odph_ipsec_alg_check(odp_ipsec_capability_t capa,
 			 odp_auth_alg_t auth_alg,
 			 uint32_t auth_key_len);
 
+/**
+ * Return the default ICV length of an algorithm
+ *
+ * IPsec API specifies default ICV length for each authentication and
+ * combined mode algorithm. This function returns the default ICV length.
+ *
+ * @param      auth_alg   Authentication algorithm
+ *
+ * @return                The default ICV length
```

Comment:
May be let's add the unit here as well, Like

"The default ICV length in bytes"